File: squad_asc_marwa_guessing game.c
#include <stdio.h>
int main(){
    int secretnum = 2;
    int guess;
    int count=0;
    int limit=3;
    int outofguesses=0;

    while(guess!=secretnum && outofguesses==0)
    {
        if(count<limit){
            printf("enter a number: ");
            scanf("%d",&guess);
            count++;
        }else {
            outofguesses=1;
        }
    }
    if(outofguesses==1){
        printf("Out of guesses");
    }else{
        printf("YOU WIN!");
    }
}

File: squad_asc_marwa_guessinggame.py
secretname = "marwa"
guess = ""
count = 0
limit = 3
flag = False

while guess != secretname and not(flag) :
    if count<limit:
        guess=input("enter a name: ")
        count = count + 1
    else:
        flag = 1


if flag:
    print("Out of guesses, YOU LOSE!")
else:
    print("YOU WIN!")
